lib/WWW/YouTube/Download.pm

```diff
@@ -166,9 +167,11 @@ sub _fetch_title {
 }
 
 sub _fetch_user {
-    my ($self, $content) = @_;
+	my ($self, $content) = @_;
 
-	if( $content =~ /<span class="yt-user-name [^>]+>([^<]+)<\/span>/ ){
+#	if( $content =~ /<span class="yt-user-name [^>]+>([^<]+)<\/span>/ ){
+#		return decode_entities($1);
```

Comment:
Rather than commenting code out, I think we're better off deleting it.  Also, for some reason this block has been indented beyond 4 spaces.